src/templates/starter-kit/store/auth/auth.state.ts

```diff
@@ -0,0 +1,45 @@
+import { Action, Selector, State, StateContext } from '@ngxs/store';
+import { Authentication } from './model/auth.model';
+import { GetAuthData } from './auth.actions';
+
+export interface AuthenticationStateModel {
+    id: string;
+    firstName: string;
+    lastName: string;
+    fullName: string;
+    email: string;
+    roles: string[];
+}
+
+@State<AuthenticationStateModel>({
+    name: 'authStateModule',
+    defaults: {
+        id: '',
+        firstName: '',
+        lastName: '',
+        fullName: '',
+        email: '',
+        roles: []
+    }
+})
+export class AuthStateModule {
```

Comment:
How far are we wanting to go with the starter kit?
We could base this off of Austin's article here: https://medium.com/@amcdnl/authentication-in-ngxs-6f25c52fd385

Reply:
It takes time. I can add this in later versions.